package.json

```diff
@@ -62,7 +62,7 @@
     "chai": "^3.5.0",
     "cookie-parser": "^1.3.4",
     "es5-shim": "^4.1.0",
-    "eslint-config-loopback": "^1.0.0",
+    "eslint-config-loopback": "^4.0.0",
```

Comment:
For now it doesn't matter but I think we should move away from having a Gruntfile at all and doing it all via NPM scripts instead. As for your current PR, you don't need to do anything.